request_a_govuk_domain/request/views.py

```diff
@@ -87,6 +93,17 @@ class RegistrarDetailsView(FormView):
     success_url = reverse_lazy("registrant_type")
     change = False
 
+    def get(self, request: HttpRequest, *args: str, **kwargs: reverse_lazy):
+        if "init" in self.request.session:
+            print("init")
+            self.request.session.pop("init")
+            self.request.session.save()
+            return super().get(request, *args, **kwargs)
+        if "Referer" in self.request.headers:
+            if "registrant-type" in self.request.headers["Referer"]:
```

Comment:
You can come to this page from back button as well as the confirmation page links, you should as the confirmation url to the allowed referrer lists too?